src/sempy_labs/tom/_model.py

```
@@ -5261,6 +5327,425 @@ def format_dax(
+
+        object_type = object.ObjectType
+
+        if object_type not in [
+            TOM.ObjectType.Table,
+            TOM.ObjectType.Column,
+            TOM.ObjectType.Measure,
+            TOM.ObjectType.Hierarchy,
+        ]:
+            raise ValueError(
+                f"{icons.red_dot} This function only supports adding synonyms for tables/columns/measures/hierarchies."
+            )
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)